### PR TITLE
chore(arm64) build Mac M1 binaries; closes #428

### LINKED ISSUE #428: Build/release arm64 binaries
See https://github.com/infracost/infracost/pull/287 for a previous attempt

### PR DESCRIPTION
**Pull request by xeger**
Still need to update the docs here (separate repo):
https://www.infracost.io/docs/#installation

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,6 +40,8 @@ release: build_all
 	cd build; tar -czf $(BINARY)-windows-amd64.tar.gz $(BINARY)-windows-amd64
 	cd build; tar -czf $(BINARY)-linux-amd64.tar.gz $(BINARY)-linux-amd64
 	cd build; tar -czf $(BINARY)-darwin-amd64.tar.gz $(BINARY)-darwin-amd64
+	cd build; tar -czf $(BINARY)-darwin-arm64.tar.gz $(BINARY)-darwin-arm64
+	cd docs/generated; tar -czvf docs.tar.gz *.md
```
Comment (alikhajeh1) [inline Makefile:44]: We removed this a while ago, maybe your fork was behind?

File: README.md [hunk 2]
```diff
@@ -23,8 +23,9 @@ Infracost shows cloud cost estimates for Terraform projects. It helps developers
     Linux/macOS manual:
     ```sh
     os=$(uname | tr '[:upper:]' '[:lower:]') && \
-    curl -s -L https://github.com/infracost/infracost/releases/latest/download/infracost-$os-amd64.tar.gz | tar xz -C /tmp && \
-    sudo mv /tmp/infracost-$os-amd64 /usr/local/bin/infracost
+    arch=$(uname -m | tr '[:upper:]' '[:lower:]' | sed -e s/x86_64/amd64/)
```
Comment (alikhajeh1) [inline README.md:26]:
```suggestion
    arch=$(uname -m | tr '[:upper:]' '[:lower:]' | sed -e s/x86_64/amd64/) && \
```

**Review by alikhajeh1 — COMMENTED** [review]
Thanks @xeger! I'll update the docs when as part of https://github.com/infracost/docs/pull/50

File: Makefile (hunk 1 above)
Comment (alikhajeh1) [inline Makefile:44]:
```suggestion
```

**Comment by alikhajeh1** [comment]
@aliscott should we merge this as part of the next version bump so the readme and the docs are in sync?

**Comment by aliscott** [comment]
@alikhajeh1 yeah let's get this in. @xeger github isn't recognising your username on the commits, not sure if this is an issue with your git/github config.

**Pull request merged**